external/fv3fit/fv3fit/train.py

```diff
@@ -53,52 +46,40 @@ def dump_dataclass(obj, yaml_filename):
 
 
 def main(args):
-    data_path = parse_data_path(args.data_path)
-    (
-        train_config,
-        train_data_config,
-        val_data_config,
-    ) = fv3fit._shared.config.load_configs(
-        args.config_file,
-        data_path=data_path,
-        output_data_path=args.output_data_path,
-        timesteps_file=args.timesteps_file,
-        validation_timesteps_file=args.validation_timesteps_file,
-    )
+    with open(args.train_config, "r") as f:
+        train_config = fv3fit.TrainingConfig.from_dict(yaml.load(f))
+    with open(args.train_data_config, "r") as f:
+        train_data_config = loaders.BatchesLoader.from_dict(yaml.load(f))
+    with open(args.val_data_config, "r") as f:
```

Comment:
I mostly changed this to simplify the interface, since my understanding was we usually have validation data. Training codes are welcome to ignore the validation data they are given.

If we want this to be optional (which is what I'm hearing) I'll turn validation data into an optional requirement. if not passed I'll set validation batches to `[]` to avoid None checks.

Reply:
I would second making it an optional argument, since I think it's more confusing to have to configure validation data for an RF routine that doesn't use it than to have to manage an optional arg.

Reply:
@brianhenn I am planning internally to make validation data a required argument for training functions, but the random forest training function can completely ignore this argument. It will be given an empty list when validation data is not passed.

Reply:
Done.